Source/UIKit/UIControl.swift

```diff
@@ -20,6 +21,21 @@ extension UIControl {
             .map { $0 as? UIControl }
             .flatMapError { _ in SignalProducer(value: nil) }
     }
+
+    /// Creates a bindable property to wrap a control's value.
+    /// 
+    /// This property uses `UIControlEvents.ValueChanged` and `UIControlEvents.EditingChanged` 
+    /// events to detect changes and keep the value up-to-date.
+    //
+    @warn_unused_result(message="Did you forget to use the property?")
+    class func rex_valueChanged<Host: UIControl, T>(host: Host, getter: Host -> T, setter: (Host, T) -> ()) -> MutableProperty<T> {
```

Comment:
I think the name `rex_valueChanged` is misleading. 

Especially considering `rex_controlEvents` above, I first assumed that this was like a pre-filtered version of `rex_controlEvents`, but then I saw thats not really what it does.

I'm not sure what would be a good name, `rex_controlEventUpdatedProperty` seems a bit clumsy.